web/src/site-admin/SiteAdminLsifJobPage.tsx

```diff
@@ -0,0 +1,166 @@
+import * as GQL from '../../../shared/src/graphql/schema'
+import AlertCircleIcon from 'mdi-react/AlertCircleIcon'
+import CheckIcon from 'mdi-react/CheckIcon'
+import ClockOutlineIcon from 'mdi-react/ClockOutlineIcon'
+import React, { FunctionComponent, useEffect, useMemo } from 'react'
+import { asError, ErrorLike, isErrorLike } from '../../../shared/src/util/errors'
+import { catchError } from 'rxjs/operators'
+import { eventLogger } from '../tracking/eventLogger'
+import { fetchLsifJob } from './backend'
+import { LoadingSpinner } from '@sourcegraph/react-loading-spinner'
+import { PageTitle } from '../components/PageTitle'
+import { sortBy } from 'lodash'
+import { RouteComponentProps } from 'react-router'
+import { Timestamp } from '../components/time/Timestamp'
+import { useObservable } from '../util/useObservable'
+
+const JobArguments: FunctionComponent<{ args: { [name: string]: string } }> = ({ args }) => (
+    <table className="w-100 mb-0 table table-sm">
+        <tbody>
+            {sortBy(Array.from(Object.entries(args)), ([key]) => key).map(([key, value]) => (
```

Comment:
`Object.entries()` returns an array.
```suggestion
            {sortBy(Object.entries(args), ([key]) => key).map(([key, value]) => (
```